docs/requirements.txt

```diff
@@ -1,3 +1,31 @@
-mkdocs-material
-mkdocstrings
-mike
+charset-normalizer==3.1.0
+click==8.1.3
+cuid==0.4
+ffmpeg-python==0.2.0
+filelock==3.11.0
+future==0.18.3
+gql==3.4.0
+graphql-core==3.2.3
+idna==3.4
+kili==2.131.1
```

Comment:
Not sure this should be hardcoded, else doc will never be generated for new versions of the sdk right ?